### PR TITLE
refactor(i18n): replace i18n-js with i18next

### PR DESCRIPTION
**Pull request by mmaciorowski-sage**
### Proposed behaviour
Use i18next on welcome page
Use i18next for the switch component
https://github.com/Sage/carbon/blob/master/rfcs/text/i18n.md

### Current behaviour
i18n-js

### Checklist

- [x] Commits follow our style guide
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [ ] Carbon implementation and Design System documentation are congruent

### Testing instructions
No test changes are required.

### CODE REVIEW

**Comment by mkrds** [comment]
All looks good but couldn't we already remove `/utils/i18n/en` file?

File: src/__experimental__/components/switch/switch-slider.component.js [hunk 1]
```diff
@@ -17,9 +18,10 @@ const SwitchSlider = (props) => {
     info,
     useValidationIcon,
   } = props;
+  const t = useTranslation();
 
-  const on = I18n.t("switch.on", { defaultValue: "ON" }).toUpperCase();
-  const off = I18n.t("switch.off", { defaultValue: "OFF" }).toUpperCase();
+  const on = t("switch.on", "ON").toUpperCase();
```
Comment (mkrds) [inline src/__experimental__/components/switch/switch-slider.component.js:23]: Is this going to work if someone uses `Switch` component that is not wrapped by the `I18nProvider`?

Comment (mmaciorowski-sage) [reply src/__experimental__/components/switch/switch-slider.component.js:23]: No, then `I18nProvider` will throw an error.

**Comment by mmaciorowski-sage** [comment]
> All looks good but couldn't we already remove `/utils/i18n/en` file?

Not yet. We need to change components gradually. At the end we will remove `i18n-js`.

File: src/__experimental__/components/switch/switch-slider.component.js (hunk 1 above)
Comment (mkrds) [reply src/__experimental__/components/switch/switch-slider.component.js:23]: I'm not sure if it's a good idea to have a component library that enforces the use of `I18nProvider`.

**Review by edleeks87 — APPROVED** [review]

**Review by mkrds — APPROVED** [review]

**Review by Alec-Walker-Sage — APPROVED** [review]

**Pull request merged**